From 21468d70f82894b29a6c91c55e6b83f1ed79c78a Mon Sep 17 00:00:00 2001
From: Marc Mao <marc.mao@intel.com>
Date: Wed, 27 Feb 2013 19:13:17 +0800
Subject: [PORT FROM MAIN]Enable framework native fence for PVR DDK1.10

BZ: 102283

This patches squash two changed together.

It enable native fence sync and wait sync together.

USE_WAIT_SYNC will use eglWaitSync to wait the producer complete
rendering. it gives better CPU/GPU overlap.

Original: Internal

Orig-Change-Id: I430b77caee0bc981a1387c13e157dd2a02608405
Category: device enablement
Domain: Graphics.3D-CTP
Origin: internal
Upstream-Candidate: no
Orig-Change-Id: I1a2e3be79aec4d4b832f5f1bf6269df2a0ffbbb9
Signed-off-by: Marc Mao <marc.mao@intel.com>

Change-Id: Ia189c6d5b41ae1a48535a2dcd83de0b13599250f
Signed-off-by: binchen1 <bin.chen@intel.com>
---
 libs/gui/Android.mk |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/libs/gui/Android.mk b/libs/gui/Android.mk
index c14c950..c2e53d5 100644
--- a/libs/gui/Android.mk
+++ b/libs/gui/Android.mk
@@ -49,6 +49,11 @@ ifeq ($(TARGET_BOARD_PLATFORM), tegra3)
 	LOCAL_CFLAGS += -DDONT_USE_FENCE_SYNC
 endif
 
+ifeq ($(TARGET_BOARD_PLATFORM), clovertrail)
+	LOCAL_CFLAGS += -DUSE_NATIVE_FENCE_SYNC
+	LOCAL_CFLAGS += -DUSE_WAIT_SYNC
+endif
+
 include $(BUILD_SHARED_LIBRARY)
 
 ifeq (,$(ONE_SHOT_MAKEFILE))
-- 
1.7.4.1

